test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildACppCliProjectWithTransitiveDeps.cs

```diff
@@ -23,7 +23,7 @@ public void It_can_generate_correct_depsJson_file()
             depsJsonContent.Should().Contain("NETCoreCppCliTestC.dll", "should contain transitive reference");
         }
 
-        [FullMSBuildOnlyFact(Skip = "https://github.com/dotnet/sdk/issues/3785")]
+        [FullMSBuildOnlyFact]
```

Comment:
Unskipping this test failed. Marking it with the successor issue. Details here: https://github.com/dotnet/sdk/issues/11008#issuecomment-2183528729